scss/_functions.scss

```
@@ -21,12 +21,12 @@
 }
 
 // Starts at zero
-// Another grid mixin that ensures the min-width of the lowest breakpoint starts at 0.
-@mixin _assert-starts-at-zero($map) {
+// Used to ensure the min-width of the lowest breakpoint starts at 0.
+@mixin _assert-starts-at-zero($map, $map-name) {
```

@mdo, we're completely covered if we add this, just in case developers also use this function in their custom code:
```suggestion
@mixin _assert-starts-at-zero($map, $map-name: "$grid-breakpoints") {
```